harvard-lts/mps-viewer
# Increasing Test Code Coverage and Fixing GitHub Actions

**Increasing Test Code Coverage and Fixing GitHub Actions.**
* * *

**JIRA Ticket**: [LTSVIEWER-279)](https://jira.huit.harvard.edu/browse/LTSVIEWER-279)

# What does this Pull Request do?
This increases the overall code coverage in the Jest tests from 77.11% to 86.95%. Additionally it fixes the GitHub Action that runs the test. The Action completes successfully. Finally, I upgraded the Checkout step of the Action to use version3, which upgrades Node to version 16. 

# How should this be tested?

A description of what steps someone could take to:
* First, have your mps-embed container running.
* Build mps-viewer using the `LTSVIEWER-279` branch.
* Bash into the mps-viewer container: `docker exec -it mps-viewer bash`
* Run all the tests: `npm run test`
* Confirm that all 18 tests pass. Confirm the code coverage for "All files" is 86.95%. 
* Confirm that the GitHub Action associated with this PR finishes successfully. (https://github.com/harvard-lts/mps-viewer/actions/runs/6238486449/job/16934409527?pr=85)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? Yes
- integration tests? Yes

# Interested parties
@f8f8ff @enriquediaz
